src/app/shared/asyncapi.service.ts

```diff
@@ -82,43 +82,47 @@ export class AsyncApiService {
         };
     }
 
-    private mapServers(servers: { [key: string]: Server }): Map<string, Server> {
+    private mapServers(servers: ServerAsyncApi["servers"]): Map<string, Server> {
```

Comment:
Wow what is this cool type definition feature called like?

Reply:
I believe this is called indexed access types
https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html